src/main/kotlin/org/jitsi/nlj/util/BitrateTracker.kt

```diff
@@ -33,5 +33,5 @@ open class BitrateTracker @JvmOverloads constructor(
     val rate: Bandwidth
         get() = getRate()
     fun update(dataSize: DataSize, now: Long = clock.millis()) = tracker.update(dataSize.bits, now)
-    fun getAccumulatedSize(now: Long = clock.millis()) = tracker.getAccumulatedCount(now).bps
+    fun getAccumulatedSize(now: Long = clock.millis()) = tracker.getAccumulatedCount(now).bits
```

Comment:
Was this a bug?

Reply:
smells like it, cumulative over a period of time must be a unit of data.

Reply:
Nothing was using it, but yes, conceptually this should be bits not bps as George said.